### PR TITLE
feat: migrated address & updated linter

### PR DESCRIPTION
**Pull request by kozlovzxc-at-lido**
### Description

* Migrated Address component to CSS Modules
* Updated eslint, prettier, typescript so it supports `satisfies` operator

### Demo

image

image

### CODE REVIEW

File: packages/data-table/DataTable.stories.tsx [hunk 1]
```diff
@@ -10,23 +10,24 @@ export default {
   },
 } as Meta
 
-export const Base: Story<DataTableProps & Pick<DataTableRowProps, 'loading'>> =
-  (props) => {
-    const { loading, ...rest } = props
+export const Base: Story<
+  DataTableProps & Pick<DataTableRowProps, 'loading'>
+> = (props) => {
+  const { loading, ...rest } = props
```
Comment (Jabher) [inline packages/data-table/DataTable.stories.tsx:16]:
```suggestion
> = ({ loading, ...rest }) => {
```

File: packages/modal/useModalFocus.ts [hunk 2]
```diff
@@ -11,7 +11,7 @@ export const useModalFocus = (): React.RefObject<HTMLDivElement> => {
 
     return Array.from(nodes).filter(
       (node): node is HTMLElement =>
-        node && node instanceof HTMLElement && node.offsetParent !== null
+        node && node instanceof HTMLElement && node.offsetParent !== null,
```
Comment (Jabher) [inline packages/modal/useModalFocus.ts:14]:
```suggestion
        node instanceof HTMLElement && node.offsetParent !== null,
```

**Review by Jabher — APPROVED** [review]

File: packages/data-table/DataTable.stories.tsx (hunk 1 above)
Comment (kozlovzxc-at-lido) [reply packages/data-table/DataTable.stories.tsx:16]: I will refactor this component later, these are just automatic lint:fix changes

File: packages/modal/useModalFocus.ts (hunk 2 above)
Comment (kozlovzxc-at-lido) [reply packages/modal/useModalFocus.ts:14]: Makes sense, will update once I get to this hook

**Pull request merged**